alacritty/src/config/bindings.rs

```diff
@@ -161,7 +161,8 @@ pub enum Action {
     /// Hide the Alacritty window.
     Hide,
 
-    /// Hide all windows except Alacritty.
+    /// Hide all windows except Alacritty on macOS.
```

Comment:
```suggestion
    /// Hide all windows other than Alacritty on macOS.
```

Reply:
Apologies, I thought you were referring to the changelog